### PR TITLE
109 filters and sort

### PR DESCRIPTION
**Pull request by mrkava**
#109 

- [ ] @igorkasyanchuk 
- [ ] @Kmalynss 
- [ ] @f1nwe 

- Adds gem smart_listing for sorting, search and filters
- Adds search, filtering and sorting for tables with data: users moderating and issue moderating; user profile page
- Adds sidebar for all issues page and category issues page with search and all categories list with issues count
- Some small improvements (additional validations in issue model; changed default image for issue etc)

### CODE REVIEW

File: app/controllers/account/admin/users_controller.rb [hunk 1]
```diff
@@ -7,7 +7,13 @@ class UsersController < ApplicationController
       before_action :admin?, only: [:change_role]
 
       def index
-        @users = User.page(params[:page]).per(20)
+        @users_scope = User.all
```
Comment (f1nwe) [inline app/controllers/account/admin/users_controller.rb]: I thing it is a bad idea to get all users. What will be if we have a lot of users? Somebody mentioned about it on last Demo. Maybe somebody from mentors can say something about  it?

File: config/locales/uk.yml [hunk 2]
```diff
@@ -1,6 +1,13 @@
 ---
 uk:
   activerecord:
+    attributes:
+      issue:
+        statuses:
+          open: 'Запит прийнято'
```
Comment (f1nwe) [inline config/locales/uk.yml]: Maybe it's better to use 'Зверення or something else. It is not 'запит''?

File: .rubocop.yml [hunk 3]
```diff
@@ -43,4 +43,10 @@ Metrics/BlockLength:
...
     - 'spec/**/*'
-    - 'config/environments/*'
\ No newline at end of file
+    - 'config/environments/*'
+
+Metrics/ClassLength:
+  Description: 'Avoid classes longer than 110 lines of code.'
+  Enabled: true
+  CountComments: false
+  Max: 110
```
Comment (mrkava) [inline .rubocop.yml]: Issue model is bigger than 100 lines now... and it looks like when i will merge your last commit will be even bigger

But we can try to move some methods to concerns...

Comment (f1nwe) [inline .rubocop.yml]: It is a bad to modify rubocop cops. Better to exclude file from it. I already done it in my last commits. =)

File: app/controllers/account/admin/users_controller.rb (hunk 1 above)
Comment (mrkava) [inline app/controllers/account/admin/users_controller.rb]: We get all and then smart_listing paginates them.

Comment (f1nwe) [inline app/controllers/account/admin/users_controller.rb]: If I`m right, firstly you get all users, and after that take only part from them. Maybe you can do this two actions in one requests. But I`m not sure. I`ve written it to draw attention of mentors, maybe somebody can give advise.

File: .rubocop.yml (hunk 3 above)
Comment (mrkava) [inline .rubocop.yml]: To exclude such important file like issue model from rubocop also not so good idea))

Comment (f1nwe) [inline .rubocop.yml]: You are right, but I said only about exluding from one cop, not from rubocop сompletely

**Pull request closed without merging**